at_secondary/at_secondary_server/lib/src/connection/outbound/outbound_message_listener.dart

```diff
@@ -76,21 +77,25 @@ class OutboundMessageListener {
         if (result.startsWith('data:') ||
             (result.startsWith('@') && result.endsWith('@'))) {
           return result;
-        } else if (result.startsWith('error:')) {
+        } else if (result.contains('errorCode')) {
```

Comment:
Are you sure there will never be any other situation other than the one you've thought about, in which a response can contain the string 'errorCode'?

Reply:
Added error: before the JSON response.